Make.inc

```diff
@@ -853,6 +858,10 @@ endif
 ifneq (,$(findstring aarch64,$(ARCH)))
 OPENBLAS_DYNAMIC_ARCH:=0
 OPENBLAS_TARGET_ARCH:=ARMV8
+ifeq ($(OS),Darwin)
+# Apple Chips are all at least A12Z
+MCPU:=apple-a12
```

Comment:
So the main issue I worry about this here is that this could override the setting of the compiler if it is ever changed to a different one.

It's certainly much less of an issue on apple platform where there are only going to be relatively few platforms to keep track of and it can still be overwritten if necessary by setting `MCPU` on the make command line so I guess this is OK...